File: Python/openstk/gl.py
import numpy as np
from OpenGL import GL as gl
from openstk.gfx import IOpenGraphicAny

# typedefs
class GLMeshBufferCache: pass
class QuadIndexBuffer: pass

# IOpenGLGraphic
class IOpenGLGraphic(IOpenGraphicAny):
    # cache
    meshBufferCache: GLMeshBufferCache
    quadIndices: QuadIndexBuffer

File: Graphics/OpenStack.Graphics.OpenGL/Gl.cs
namespace OpenStack.Graphics.OpenGL
{
    /// <summary>
    /// IOpenGLGraphic
    /// </summary>
    public interface IOpenGLGraphic : IOpenGraphicAny<object, GLRenderMaterial, int, Shader>
    {
        // cache
        public GLMeshBufferCache MeshBufferCache { get; }
        public QuadIndexBuffer QuadIndices { get; }
    }
}